scripts/build_win.sh

```diff
@@ -8,20 +7,22 @@ shift
 export AZURE_STORAGE_ACCESS_KEY=$1
 shift
 export AZURE_STORAGE_ACCOUNT=$1
+shift
+export TRAVIS_TAG=$1
 
 export AZURE_CONTAINER="device-api-usb"
 
 source /c/ProgramData/nvs/nvs.sh
-nvs use 10.12.0
+nvs use 11.5.0
 
 git submodule update --init --recursive
 
 # source activate py27
 
 npm --add-python-to-path='true' --debug --global --production windows-build-tools
-npm install
+npm ci
 
-ARCH=x64 npm run napi
-ARCH=ia32 npm run napi
+ARCH=x64 npm run napi-64
+ARCH=ia32 npm run napi-32
```

Comment:
Usage of env variables inside package.json script commands does not really work on windows! Therefore, i created two separate command for building napi on x64 and ia32 architectures.